### PR TITLE
page layout

### PR DESCRIPTION
**Pull request by IgorSinyak**
HTML + SCSS

### CODE REVIEW

File: scss/components/mixin.scss [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    text-align: center;
+
+    &:hover {
+      background-color: lighten($color, 20%);
+      transition: 0.2s;
+    }
+  }
+}
+
+
```
Comment (vitali-usik) [inline scss/components/mixin.scss]: Extra line

File: scss/components/mixin.scss [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+@mixin option-column() {
+  @include flexbox;
+  width: 25%;
+  height: 377px;
+  border-top: 7px solid;
+}
+
+@mixin option-column-colors($bg-color) {
+    background-color: $bg-color;
+  @if $bg-color == $color-green-blue {
```
Comment (vitali-usik) [inline scss/components/mixin.scss:36]: Do we need braces for condition?

File: scss/components/mixin.scss [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    border-color: #fff;
+    color: #fff;
```
Comment (vitali-usik) [inline scss/components/mixin.scss:38]: Why do you use var for green-blue and not for #fff?

File: scss/components/reset.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+li {
+  list-style-type: none;
+}
+
+input,
+textarea,
+button {
+  outline: none;
+  &:focus::-webkit-input-placeholder {
+    color:transparent;
```
Comment (vitali-usik) [inline scss/components/reset.scss:23]: Why do you use transparent color?

File: index.html [hunk 5]
```diff
@@ -6,6 +6,761 @@
...
+          </li>
+          <li>
+            <h5>Custom Backgrounds</h5>
+            <p>All our modules are built with the possi bility to thse choose between different design and you change.</p>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <div class="about-company">
+      <div class="container">
```
Comment (vitali-usik) [inline index.html]: duplicated class name

File: index.html [hunk 6]
```diff
@@ -6,6 +6,761 @@
...
+          <div class="tab-menu">
+            <ul>
+              <li><a href="#">OUR HISTORY</a></li>
+              <li><a href="#">OUR BIOGRAPHY</a></li>
+              <li><a href="#">OUR SKILLS</a></li>
+            </ul>  
+          </div>
+          <div class="skills">
+            <ul>
+              <li>#
```
Comment (vitali-usik) [inline index.html]: Typo?

File: index.html [hunk 7]
```diff
@@ -6,6 +6,761 @@
...
+            <div class="mini-author-photo">
+              <div class="mini-author-photo-wrapper"></div>
+            </div>
+            <div class="mini-author-photo">
+              <div class="mini-author-photo-wrapper"></div>
+            </div>
+            <div class="mini-author-photo">
+              <div class="mini-author-photo-wrapper"></div>
+            </div>
+            <div id="right-arrow">></div>
```
Comment (vitali-usik) [inline index.html]: Exta `>`

File: index.html [hunk 8]
```diff
@@ -6,6 +6,761 @@
...
+        </div>
+        <div class="flicke-feed">
+          <h5>Flicke Feed</h5>
+          <ul>
+            <li></li>
+            <li></li>
+            <li></li>
+            <li></li>
+            <li></li>
+            <li></li>
```
Comment (vitali-usik) [inline index.html:754]: Is some reason to leave these tags empty?

**Review by vitali-usik — CHANGES_REQUESTED** [review]

File: scss/components/mixin.scss (hunk 3 above)
Comment (IgorSinyak) [reply scss/components/mixin.scss:38]: because #fff is easy to write

File: scss/components/mixin.scss (hunk 2 above)
Comment (IgorSinyak) [reply scss/components/mixin.scss:36]: dont need

File: index.html (hunk 8 above)
Comment (IgorSinyak) [reply index.html:754]: it`s only container for picture, i can add empty "img" in it with any "alt"

File: index.html (hunk 7 above)
Comment (IgorSinyak) [reply index.html]: it`s an arrow symbol ))

File: index.html (hunk 6 above)
Comment (IgorSinyak) [reply index.html]: yep

File: scss/components/reset.scss (hunk 4 above)
Comment (IgorSinyak) [reply scss/components/reset.scss:23]: any other way to hide placeholder on focus?

File: index.html (hunk 5 above)
Comment (IgorSinyak) [reply index.html]: don`t understand what you mean

File: scss/components/mixin.scss (hunk 3 above)
Comment (vitali-usik) [reply scss/components/mixin.scss:38]: Please read about consistency

File: scss/components/reset.scss (hunk 4 above)
Comment (vitali-usik) [reply scss/components/reset.scss:23]: Do you need to hide placeholder on focus? Or when you're starting typing?

File: index.html (hunk 5 above)
Comment (vitali-usik) [reply index.html]: You have another class-name 'container'. A little bit cofusing

File: index.html (hunk 7 above)
Comment (vitali-usik) [reply index.html]: Please use for it Escape character

File: index.html (hunk 8 above)
Comment (vitali-usik) [reply index.html:754]: But if you needn't such elements - you can remove it

File: scss/components/reset.scss (hunk 4 above)
Comment (IgorSinyak) [reply scss/components/reset.scss:23]: dont know, i think hide on focus is better

File: index.html (hunk 5 above)
Comment (IgorSinyak) [reply index.html]: maybe, i`ll check

**Review by vitali-usik — APPROVED** [review]

**Pull request merged**